<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAverageDeviation</class>
 <widget class="QWidget" name="FormAverageDeviation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Measure of central tendency:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="scalingFactor">
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Scaling Factor:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="tendencyCombo">
     <item>
      <property name="text">
       <string>Mean</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Median</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mode</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="roundingLabel">
     <property name="text">
      <string>Round Elevation Values to Nearest:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="roundingValue">
     <item>
      <property name="text">
       <string>0.1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.001</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.0001</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.00001</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.000001</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
